public-ui/src/app/data-browser/views/survey-view/survey-view.component.html

```diff
@@ -374,7 +380,7 @@ <h6 class="sub-question-text" *ngIf="sqa.stratum4 !== subQuestion2.conceptName">
                                     </div>
                                   </div>
                                   <div class="survey-tbl-exp-r" *ngFor="let sqa2 of subQuestion2.countAnalysis.results"
-                                    (click)="showAnswerGraphs(sqa2, subQuestion2, 2, $event)" #subTable2>
+                                       (click)="showAnswerGraphs(sqa, subQuestion, 2, $event)" #subTable1>
```

Comment:
Should we be sending sqa2 and subquestion2 in here ?

Reply:
I just merged with master. I think it took care of this